### ISSUE TITLE
Passing `showEditButton=true` to component doesn't work

### ISSUE THREAD

**Issue by kushdilip** [swastik/ember-inline-edit#5]
Clicking on edit icon executes `startEditing` but right after that `_handleClick` catches the event and the  required conditions fails hence it emits `close` event. 

Solution is to modify `startEditing` method as below

```diff
-  startEditing () {
+  startEditing(event) {
      info('[ember-inline-edit] Got the `startEditing` action')
      Ember.run(this, function(){ set(this, 'isEditing', true) })
      this._focusOnInput()
+     event.stopPropagation();
    },
```

**Comment by swastik** [comment]
Thanks for reporting! 😄 

I'll get this fixed.

**Comment by swastik** [comment]
This is fixed now in the 0.1.6 release! Thanks once again for reporting! 👍
